modules/olm-restricted-networks-configuring-operatorhub.adoc

```diff
@@ -54,13 +54,20 @@ $ oc adm catalog mirror \
     <registry_host_name>:<port>/olm/redhat-operators:v1 \//<1>
     <registry_host_name>:<port> \
     [-a ${REG_CREDS}] \//<2>
-    [--insecure] <3>
+    [--insecure] \//<3>
+    [--filter-by-os="<os>/<arch>"] <4>
 
 mirroring ...
 ----
 <1> Specify your Operator catalog image.
-<2> Optional: If required, specify the location of your registry credentials file.
-<3> Optional: If you do not want to configure trust for the target registry, add the `--insecure` flag.
+<2> Optional: If required, specify the location of your registry credentials
+file.
+<3> Optional: If you do not want to configure trust for the target registry, add
+the `--insecure` flag.
+<4> Optional: To mirror a smaller set of images for any multi-arch images that might
```

Comment:
I don't think I'd say "multi-arch" because I think that's internal jargon. I might say something like:

> Because the registry contains some images that are valid for only one architecture, you can specify the architecture that your cluster runs on to download only the images that are valid for it.